platform/view/services/db/driver/sql/unversioned.go

```diff
@@ -50,15 +50,15 @@ func (db *Unversioned) SetState(ns, key string, val []byte) error {
 
 		_, err := db.txn.Exec(query, val, ns, key)
 		if err != nil {
-			return fmt.Errorf("could not set val for key [%s]: %w", key, err)
+			return fmt.Errorf("could not set val for key [%s]: %w", key, mapError(err))
```

Comment:
given that we are changing this, can we switch to `errors` rather than using fmt.Errorf?